robustbench/model_zoo/cifar100.py

```diff
@@ -381,6 +397,10 @@ def forward(self, x):
          lambda: WideResNet(num_classes=100, depth=34, sub_block1=True),
          'gdrive_id': '1-9GAld_105-jWBLXL73btmfOCwAqvz7Y'
     }),
+    ('Modas2021PRIME_RN18_100', {
```

Comment:
and also i'd remove here `_100` and in the json name as well (for the same reason as above).

other than that, looks good to me! thanks a lot!

Reply:
Updated, let me know if somethings missing.